{
    "affected": [
        {
            "package": {
                "ecosystem": "Photon OS:3.0",
                "name": "runc",
                "purl": "pkg:rpm/vmware/runc?distro=photon-3"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "1.0.0.rc93-2.ph3"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:3.0",
                "name": "python-cryptography",
                "purl": "pkg:rpm/vmware/python-cryptography?distro=photon-3"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "2.8-2.ph3"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:3.0",
                "name": "tar",
                "purl": "pkg:rpm/vmware/tar?distro=photon-3"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "1.30-5.ph3"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2021-3.0-0239",
    "modified": "2023-05-15T05:25:03Z",
    "published": "2021-05-19T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-3.0-239"
        }
    ],
    "related": [
        "CVE-2021-30465",
        "CVE-2021-20193",
        "CVE-2020-36242"
    ]
}